.github/workflows/publish-preview.yml

```diff
@@ -0,0 +1,21 @@
+name: Publish GitHub Actions and CircleCI Orb
+
+on:
+  push:
+    paths:
+      - 'circleci-orbs/preview.yml'
+    branches:
+      - master
+
+jobs:
+  publish_circleci_orb:
+    name: Publish CircleCI Orb
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: circleci-orb
+      - run: sudo snap install circleci
+      - run: circleci orb validate ./circleci-orbs/preview.yml --host https://circleci.com --token ${{ secrets.CIRCLE_TOKEN }}
+      - run: circleci orb publish ./circleci-orbs/preview.yml architect/preview@dev:master --host https://circleci.com --token ${{ secrets.CIRCLE_TOKEN }}
```

Comment:
How come we publish dev:master and not for new cli tags?

Reply:
David wanted us to only publish new versions when this file was changed